docs/source/docker/advanced/object_level.rst

```diff
@@ -0,0 +1,127 @@
+predictions. Make sure that you upload object detection predictions and register
+them in the `tasks.json` file.
+
+
+Selection on Object Level
+-------------------------
+Once you have everything set up as described above, you can run selection on
+object level by setting the `object_level.task_name` argument in the Lightly
+docker config. The argument should be set to the task name you used for your 
+predictions. If you uploaded the predictions to `.lightly/predictions/vehicles_object_detections`
+then you should set `object_level.task_name` to `vehicles_object_detections`.
+
+
+Here are examples on how you can schedule an object level job, either in the 
+Lightly Web App or from python code.
+
+.. tabs::
+
+    .. tab:: Web App
+
+        **Trigger the Job**
+
+        To trigger a new job you can click on the schedule run button on the dataset
+        overview as shown in the screenshot below:
+
+        .. figure:: ../integration/images/schedule-compute-run.png
+
+        After clicking on the button you will see a wizard to configure the parameters
+        for the job.
+
+        .. figure:: ../integration/images/schedule-compute-run-config.png
+
+        In this example we have to set the `object_level.task_name` parameter 
+        in the docker config, all other settings are default values. The 
+        resulting docker config should look like this:
+
+        .. literalinclude:: code_examples/object_level_worker_config.txt
+            :caption: Docker Config
+            :language: javascript
+
+        The Lightly config remains unchanged.
+
+    .. tab:: Python Code
+
+        .. literalinclude:: code_examples/python_run_object_level.py
+
+
+Object Crops Dataset
+--------------------
+Once the docker job is started it fetches all images and predictions from the 
+remote datasource and processes them. For each prediction, the docker crops
+the object from the full image and creates an embedding for it. Then it selects
+a subset of the objects and uploads **two** datasets to the Lightly Platform:
+
+1. The crops and embeddings of the selected objects are uploaded to a new
+   object *crops* dataset on the platform. The dataset has the same name as the
+   original image dataset but with a "-crops" suffix appended to it.
+2. If an object is selected, then the full image containing that object is 
+   also uploaded. You can find these images in the original dataset from which
+   you started the selection job.
+
+You can see example images of the two datasets below.
+
+Object Crop Dataset:
+
+.. figure:: images/object_level_vehicle_crops_examples.jpg
+    
+    
+Original Full Image Dataset:
+
+.. figure:: images/object_level_vehicle_examples.jpg
+
+
+Analyzing the Crop Dataset
+--------------------------
+
+The crop dataset allows you to analyze your data on an object level. If we go
+to our crops dataset and select the *Embedding* view in the menu we can see that
+different types of vehicles are grouped together.
```

Comment:
Maybe add "what we are looking for" and how the analysis helps us?

E.g. (just dumping ideas here) we are interested in understanding the diversity of cars in our dataset. Do we have have images of cars from different weather conditions? Do we have also vans?

What sort of special trucks do we have? Do we have too many school buses?